snappi_ixnetwork/ngpf.py

```diff
@@ -265,6 +265,15 @@ def _configure_ethernet(self, ixn_parent, ethernet, ixn_dg):
             ixn_ethernet.VlanCount = len(ethernet.vlans)
             ixn_ethernet.EnableVlans.Single(ixn_ethernet.VlanCount > 0)
             self._configure_vlan(ixn_ethernet.Vlan, ethernet.vlans)
+        if (
+            ethernet.get("ipv4") is not None
```

Comment:
According to discussion. Please check if we reduced access of find() operation using self.select_node(ixn_ethernet.href, children=["ipv4", "ipv6"])

Reply:
didn't observe difference for 1 device, but observed 1s difference for 10 devices. So, added additional check